Added section on storing device frames within a project

mark5422 opened the pull request:
Added a section on storing device frames within a project and the possible copyright implications of such.

Commit:
Added section on storing device frames within a project

Added a section on storing device frames within a project and the possible copyright implications of such.

KrauseFx commented on README.md:
````diff
@@ -244,6 +244,10 @@ Check out the [MindNode example project](https://github.com/fastlane/examples/tr
 
+## Alternative location to store device_frames
+
+Device frames can also be stored in a ```./fastlane/screenshots``` directory if you prefer rather than in the ```.frameit``` directory. If doing so please be aware that Apple's images are copyrighted and should not be redistributed as part of a repository so you may want to include them in your .gitignore file.
````
Isn't the path `./fastlane/screenshots/device_frames`?

Commit:
Added section on storing device frames within a project

Changed to say the folder is device_frames and not just the path to where it would be.

Commit:
Merge pull request #1 from TheInfiniteKind/mark5422-patch-1

Added section on storing device frames within a project

mark5422 commented:
Fixed the missing device_frames

KrauseFx commented:
Perfect, thanks for your contribution @mark5422 :+1:
